src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <div className="movies">
+    {movies.map((movie) => (
+      <MovieCard movie={movie} key={movie.imdbId} />
+    ))}
+  </div>
```

Comment:
Use semantic tags for list rendering like `<ul>` and `<li>`
Use this [video](https://mate.academy/learn/react/rendering-lists#/video/809) to split this list into component

Reply:
I've applied <ul>, <li> tags, but now the movies cards have different height. Should I write styles to correct this or there are no such requirements?